src/contextMenu.component.ts

```diff
@@ -46,11 +46,18 @@ export interface MouseLocation {
         <li *ngFor="let menuItem of visibleMenuItems" [class.disabled]="!isMenuItemEnabled(menuItem)"
             [class.divider]="menuItem.divider" [class.dropdown-divider]="useBootstrap4 && menuItem.divider"
             [attr.role]="menuItem.divider ? 'separator' : undefined">
-          <a *ngIf="!menuItem.divider" href [class.dropdown-item]="useBootstrap4"
+          <a *ngIf="!menuItem.divider && !menuItem.passive" href [class.dropdown-item]="useBootstrap4"
             [class.disabled]="useBootstrap4 && !isMenuItemEnabled(menuItem)"
             (click)="menuItem.triggerExecute(item, $event); $event.preventDefault(); $event.stopPropagation();">
+
             <template [ngTemplateOutlet]="menuItem.template" [ngOutletContext]="{ $implicit: item }"></template>
           </a>
+
+          <span style="display: block; padding: 3px 20px; clear: both; font-weight: normal; line-height: @line-height-base; white-space: nowrap;" *ngIf="!menuItem.divider && menuItem.passive" [class.dropdown-item]="useBootstrap4"
```

Comment:
I'd prefer putting a `passive` class on the `span` and putting the styles in the `styles` attribute of the `@Component` decorator.  Makes it a little cleaner.